src/components/Avatar/Avatar.css

```diff
@@ -0,0 +1,18 @@
+.Avatar {
```

Comment:
please re-order the `@apply` to follow https://github.com/kids-first/kf-uikit#using-tailwind-apply-helper